### PR TITLE
TS add speak btn for admin flashcard

### PR DESCRIPTION
**Pull request by mugetsu0422**

### CODE REVIEW

File: views/vwAdmin/wordPreview.hbs [hunk 1]
```diff
@@ -11,6 +11,24 @@
...
+    // Init SpeechSynth API
+    const synth = window.speechSynthesis;
+    $('#btn-speak').on('click', function () {
+      speak(wordname)
+    })
+    const speak = (text) => {
+      const speakText = new SpeechSynthesisUtterance(text);
+      // Get the voice by name
+      const voices = synth.getVoices();
+      const voice = voices.find(voice => voice.name === 'Microsoft David - English (United States)');
```
Comment (quytrungg) [inline views/vwAdmin/wordPreview.hbs:26]: tạo constant cho microsoft đi ông

**Review by quytrungg — APPROVED** [review]
💯

File: views/vwAdmin/wordPreview.hbs [hunk 2]
```diff
@@ -11,6 +11,24 @@
...
     })
+
+    const wordname = {{{ wordname }}}
+
+    // Init SpeechSynth API
+    const synth = window.speechSynthesis;
+    $('#btn-speak').on('click', function () {
+      speak(wordname)
+    })
+    const speak = (text) => {
```
Comment (quytrungg) [inline views/vwAdmin/wordPreview.hbs:22]:
```suggestion
    const speak = text => {
```

File: views/vwAdmin/wordPreview.hbs [hunk 3]
```diff
@@ -11,6 +11,24 @@
     $('#btn-back').on('click', function() {
       window.location.href = '/admin/topicdetail/' + {{{ topicid }}}
     })
+
+    const wordname = {{{ wordname }}}
+
+    // Init SpeechSynth API
+    const synth = window.speechSynthesis;
+    $('#btn-speak').on('click', function () {
```
Comment (quytrungg) [inline views/vwAdmin/wordPreview.hbs:19]:
```suggestion
    $('#btn-speak').on('click', () => {
```

Comment (quytrungg) [reply views/vwAdmin/wordPreview.hbs:19]: ref at here: [function vs arrow function](https://medium.com/geekculture/regular-vs-arrow-function-1f8140fbcece)

**Review by HuynhNguyen0811 — APPROVED** [review]

**Review by quytrungg — APPROVED** [review]

**Pull request merged**